lib/render.js

```diff
@@ -136,6 +136,17 @@ function generateSchema(value, tree, properties, ignoreDefault = false) {
       if (value.nullable) {
         properties[tree[i]].nullable = true;
       }
+      if (value.type == 'array') {
+        // The last element of the tree is an array. It is a plain or empty array since if not, the tree would have more elements
+        if (value.value == null || value.value.length === 0) {
+          // When it is an empty array
+          properties[tree[i]].items = {
+            type: "", //TODO: how do we know the type in empty arrays?
```

Comment:
@miguelaeh Sorry for raising the question now, I just noticed 😃 
What is the goal of explicitly setting the type of array items to empty? This outputs e.g.
```yaml
"imagePullSecrets": {
    "type": "array",
    ...
    "items": {
        "type": ""
    }
}
```

Why not just not adding the `items` property for those cases?
I can't see the advantage of explicitly setting it, but maybe there is a reason.
Thanks!

Reply:
It comes from this issue #34. Maybe @antgamdia could give us more details if it is still required

Reply:
Hi @castelblanque ,

the crux here is dealing with nullable params; when `null` has an explicit meaning in the model we have to still be able to generate a syntactically correct schema. See this excerpt:

````
"defaultPrereleasesVersionSelection": {
    "type": "array", 
     // missing 'items' property
    "description": "Default policy for allowing prereleases containing one of the identifiers",
    "default": "null",
    "nullable": true
},`
````

This is not an openapi v3.0.X valid schema (type "array" must have an `items` prop), because of the lack of the `items` property, AFAIK, from memory. To define this property, we need to pass the type of its items. If it is null, we don't have info about the item's type.
Note we are using openapi 3.0.x and the `nullable` prop has a slightly different meaning in 3.1.X.